php-frontend/src/test/java/org/sonar/php/symbols/ClassSymbolIndexTest.java

```diff
@@ -229,6 +229,7 @@ void unknownClassAndMethod() {
     assertThat(methodSymbol).isInstanceOf(UnknownMethodSymbol.class);
     assertThat(methodSymbol.visibility()).isEqualTo(Visibility.PUBLIC);
     assertThat(methodSymbol.owner()).isInstanceOf(UnknownClassSymbol.class);
+    assertThat(methodSymbol.isTestMethod().isTrue()).isFalse();
```

Comment:
The class symbol test should not test the property of the method. This test should be in a method symbol test

Reply:
Forgot to delete it with the others, thanks for catching it